app/test/frontend/golden/pkg_index_page.html

```diff
@@ -78,9 +78,10 @@ <h1 class="_visuallyhidden">Dart pub</h1>
   
 <div class="listing-sort-header">
   <div class="tooltip-base">
-    <span class="tooltip-dotted">Sorted by:</span> overall score
+    <span class="tooltip-dotted">Sorted by:</span>
+    <span id="sort-control" data-sort="default">default</span>
     <div class="tooltip-content tooltip-bottom-left">
-      Packages are sorted by the overall score. More information on <a href="/help#ranking">ranking</a>.
+      Packages are sorted by the combination of text match, overall score and platform context. More information on <a href="/help#ranking">ranking</a>.
```

Comment:
The pkg page shouldn't be sorted by text match, right?

Reply:
Text match will be applied only if there is text and ordering is either `default` or `text`. I've kept it in the tooltip because it was easier to have one tooltip for both cases that handling the case where we are on `default` with text and without text.